packages/create-project/src/helpers/git.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-empty */
+import { execSync } from 'child_process';
+import path from 'path';
+import rimraf from 'rimraf';
+
+function isInGitRepository(): boolean {
+  try {
+    execSync('git rev-parse --is-inside-work-tree', { stdio: 'ignore' });
```

Comment:
Does this return `true` if I have a `.git` directory in my home folder and I try to create a project in `~/my-app`?

Reply:
Yes, in which case we won't `git init` for them. Don't see a way to differentiate between home folder case and adding temporal inside an existing code repo.

Reply:
As discussed f2f, let's prompt the user for creating the git directory and overwriting an existing directory.

It might also be worth adding a flag to prevent creating the git repository for non-git users.